qatzip/compress.go

```diff
@@ -340,20 +342,24 @@ func (z *Writer) compressWrite(p []byte) (n int, err error) {
 		z.traceLogf(Med, "[write->qat] r:%v i:%v o:%v ibofs:%v obl:%v err:%v", remainder, in, out, consumed, z.outputBuf.Len(), err)
 
 		if err != nil {
-			if err == ErrBuffer {
-				// expand output buffer
-				// TODO grow to a maximum size
+			if err == ErrBuffer && !isPartialJob {
```

Comment:
I don't think we need to change this. I have only seen ErrBuffer in the case where the Output Buffer is too small in compression.